docs/pages/install/windows.mdx

```diff
@@ -0,0 +1,82 @@
+# Install Pelican on Windows
+
+This document explains how to install Pelican on Windows.
+
+## Install Pelican as a standalone binary
+
+Pelican provides a binary file instead of a `MSI` installer for Windows. You need to download and extract the binary and add the binary to your `PATH` environment variable.
+
+1. Navigate to [Pelican download page](../install.mdx#download-pelican-binary) and select the Pelican Windows binary.
+
+2. In the **Operating System** section, select **Windows**. Copy the link to `pelican_Windows_x86_64.zip`.
+
+3. Download the file to your computer and extract the binary from the `.zip` file.
+
+4. You may run the binary in the extracted folder, but it is recommended that you add Pelican binary to your `PATH` environment variable to allow `pelican` to be called directly from your PowerShell. To run Pelican binary from the extracted folder:
+
+    ```bat
+    rem Go to the downloaded Pelican folder
+    $ cd D:\Downloads\pelican_Windows_x86_64
+
+    rem Run Pelican binary
+    $ pelican.exe --version
+
+    Version: 7.5.8
+    Build Date: 2024-03-01T18:13:00Z
+    Build Commit: d260a07d3b057d19b7fdd36125f91a8768531258
+    Built By: goreleaser
+    ```
+
+## Add Pelican binary to your `PATH`
+
+* Add Pelican binary to your `PATH` for the current PowerShell
+
+  ```bat
+  rem Go to the binary folder
+  $ cd pelican_Linux_arm64
+
+  rem Add current folder to the PATH
+  $ set PATH=%PATH%;%CD%
+
+  rem Run Pelican binary
+  $ pelican --version
+
+  Version: 7.5.8
+  Build Date: 2024-03-01T18:13:00Z
+  Build Commit: d260a07d3b057d19b7fdd36125f91a8768531258
+  Built By: goreleaser
+  ```
+
+* Add Pelican binary to your `PATH` permanently
+
+   1. **Open System Properties**:
+      * Press `Win + R`, type `sysdm.cpl`, and press `Enter`.
+
+   2. **Access Environment Variables**:
+      * In the System Properties window, go to the `Advanced` tab.
+      * Click on the `Environment Variables` button.
+
+   3. **Edit the PATH Variable**:
+      * In the Environment Variables window, under the `System variables` section, scroll down and find the `Path` variable, then select it and click `Edit`.
+      * In the Edit Environment Variable window, click `New` and add the path to the directory containing `pelican.exe` (e.g., `C:\path\to\directory`).
+      * Click `OK` to close all windows.
+
+   4. Verify Pelican is added to `PATH`
+      * In a new PowerShell window, run the following command:
+
+        ```bat
+        rem Run Pelican binary
+        $ pelican --version
```

Comment:
Should this be ?

```suggestion
        $ pelican.exe --version
```

Reply:
I'm fairly sure you won't need to add `.exe` if the executable is already in the `PATH`, I'll double check once I get access to my Windows machine

Reply:
I verified that if I added pelican to `PATH`, I can evoke pelican by either `pelican` or `pelican.exe`. I'll leave this as-is here to be consistent with the previous instructions.